### DISCUSSION TITLE
RASPBERRY PI

### DISCUSSION THREAD

**Discussion by frenrude** [Audiveris/audiveris#453] [Ideas]
Please can you add support for Raspberry Pi for the 5.2 release?
This is rather urgent because many musicians are discovering the Raspberry Pi.
It can run Ubuntu / Debian and all dependencies (Leptonica, Tesseract4, Java11) are in the official repositories.
It just needs to be properly configured.
Thank you so much!

**Comment by maximumspatium** [comment]
> It just needs to be properly configured.

Well, I'd suggest you go ahead and try to run Audiveris in your Raspberry Pi. Detailed instructions on how to build this project are found [here.](https://github.com/Audiveris/audiveris/wiki/Installation)

This is basically a process of trial and error consisting of the following steps:

1. build the software. If it works you're done
2. if the build fails, figure out why
3. fix the build
4. return to step 1 until it succeed

We cannot do it for you because we have no access to the corresponding hardware. But we can help you with the steps 2 and 3.
